public/src/reducers/campaign.jsx

```diff
@@ -1,12 +1,14 @@
 export const defaultCampaigns = {
   all_campaigns: [],
   current_campaign: {},
+  current_campaign_metrics: {},
   has_user_joined_campaign: false
 };
 
 export const CLEAR_CAMPAIGNS = 'CLEAR_CAMPAIGNS';
 export const SET_CAMPAIGNS = 'SET_CAMPAIGNS';
 export const SET_CAMPAIGN_CURRENT = 'SET_CAMPAIGN_CURRENT';
+export const SET_CAMPAIGN_CURRENT_METRICS = 'SET_CAMPAIGN_CURRENT_METRICS';
```

Comment:
should have a clear campaign metrics action that clears this state from the store, similar to clear campaigns above